apps/accounts/models.py

```diff
@@ -270,14 +270,26 @@ def save(self, **kwargs):
         expires = now + timedelta(days=settings.SIGNUP_TIMEOUT_DAYS)
         self.expires = expires
 
-        # send an email with reset url
-        send_activation_key_via_email(self.user, self.key)
+        # send an email with activation url
+        activation_link = '%s%s' % (get_hostname(),
+                                    reverse('activation_verify',
+                                            args=(self.key,)))
+
+        mailer = Mailer(subject='Verify Your Blue Button 2.0 Developer Sandbox Account',
+                        template_text='email-activate.txt',
+                        template_html='email-activate.html',
+                        to=[self.user.email, ],
+                        context={"ACTIVATION_LINK": activation_link})
+        mailer.send()
+        logger.info("Activation link sent to {} ({})".format(self.user.username,
+                                                             self.user.email))
         super(ActivationKey, self).save(**kwargs)
```

Comment:
Mail action should come after save to avoid the condition where a save fails for some reason resulting in a user having an unusable key.